### PR TITLE
Fix dependabot example auto-label

### LINKED ISSUE #23568: Dependabot example auto-label does not work
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

Example from:

https://docs.github.com/en/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions#label-a-pull-request

Problem:
Example fails because token for permission is missing.

### What changes are you suggesting?

Add missing:

> GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Rotzbua**
### Why:

Example fails because token for permission is missing.

Closes #23568

### What's being changed (if available, include any code snippets, screenshots, or gifs):

* Fix missing `GITHUB_TOKEN` to get permission for adding label

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
@Rotzbua Thank you for opening a PR and linking it to your issue! ✨

File: content/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions.md [hunk 1]
```diff
@@ -357,6 +357,7 @@ jobs:
         run: gh pr edit "$PR_URL" --add-label "production"
         env:
           PR_URL: ${{github.event.pull_request.html_url}}
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
```
Comment (rachaelrenk) [inline content/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions.md:360]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

File: content/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions.md [hunk 2]
```diff
@@ -389,6 +390,7 @@ jobs:
...
```
Comment (rachaelrenk) [inline content/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions.md:393]:
```suggestion
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Review by rachaelrenk — DISMISSED** [review]
Hi @Rotzbua 👋 Thank you for catching this and opening a PR with updates. I made a small stylistic tweak for consistency, and will get this approved and merged! 🛸

**Review by rachaelrenk — APPROVED** [review]

File: content/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions.md (hunk 1 above)
Comment (JacobKlipper) [reply content/code-security/dependabot/working-with-dependabot/automating-dependabot-with-github-actions.md]: `thank you`

**Pull request merged**